datasets/.gitignore

```
@@ -12,6 +12,9 @@
 /titanic_wrong_number_columns.csv
 /titanic_wrong_passengerID.csv
 
+# lungs ultrasound
+/lungs ultrasound/
```

```suggestion
/lus_covid/
```
To keep it similar to the task name and not use whitespace in the file system :)